ganga/GangaCore/Utility/feedback_report.py

```diff
@@ -23,15 +23,9 @@ def deny_modification(name, x):
         pass
 _initconfigFeed()
 
-def send_request(url,data,files):
-    response = requests.post(url, data=data, files=files)
-    return response
 def report(job=None):
```

Comment:
I would like to see this changed such that the user can get report in different locations (LocalFile, DiracFile, GoogleFile) in addition to the default upload mechanism of a GoogleFile. So maybe the following signatures.
```
report()
report(job=jobs[47])
report(filetype=LocalFile)
```
The docstring should be updated to reflect this.

Finally the report function should return the file object created such that I can do
```
myreport = report()
```

Reply:
Do you want it that like: 
1. if the user uses `GoogleFile`, the report is uploaded to his google drive.
2. if the user uses `LocalFile`, the report is saved as a LocalFile on the user's disk
.... ?

Reply:
Yes, that is right. We have other file types as well that it in principle should work with. I feel that our abstract interface for IGangaFile is a bit weak though in how to implement this well. That can be the source of another issue though.